services/integration/src/models/constitution/processes.model.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const processesSchema = new Schema({
```

Comment:
This should named as ProcessSchema, since you are defining a single object not a set of objects.

Reply:
i changed name´s schema